src/app/nav/header-types.demo.html

```diff
@@ -6,7 +6,7 @@
 
 <div class="clr-example">
     <div class="main-container">
-        <header class="header-6 space-below">
+        <header class="header-1 space-below">
```

Comment:
This file was for displaying the different sections in the headers. We kept the color same so that we could focus on the sections. There is a separate demo for header colors. Up to you if you want to revert this back but wanted to let you know what this file intended to do.

Reply:
I saw the `header-colors` demo later on and meant to put this one back. 
I'll revert it.